src/components/CalendarBody.tsx

```diff
@@ -61,11 +61,12 @@ interface CalendarBodyProps<T extends ICalendarEventBase> {
   onLongPressCell?: (date: Date) => void
   onPressCell?: (date: Date) => void
   onPressEvent?: (event: T) => void
+  onSwipeHorizontal?: (d: HorizontalDirection) => void
```

Comment:
I couldn't understand why this line is inserted - is it really needed?

Reply:
Fixed! Not really sure about why I added it to be honest 😅

Reply:
Thanks! I'll take a look soon